src/Steam.package/STMemoryStoreTest.class/instance/setUp.st

```diff
@@ -0,0 +1,14 @@
+initialization
+setUp
+	| app |
+	app := STMemoryStore createAppWithClass: STExampleApp.
+	store := app store.
+	gameCat := (STGameCategory onSteamApplication: app)
+		title: 'Test Category';
+		yourself.
+	game := (STExampleGame onSteamApplication: app)
+		category: gameCat;
+		title: 'Test Game';
+		rating: 10.
+	self assert: (store query: game class with: game title) size equals: 0.
+	self assert: (store query: gameCat class with: gameCat title) size equals: 0
```

Comment:
Putting assertions in the setUp is strange.

What I would expect in here is that we create the database here.

Reply:
And there is a missing  `super tearDown` too!